### PR TITLE
More room entry type checking.

### PR DESCRIPTION
**Pull request by cyburns**
# Description

Please describe the issue of the pull request and the changes

Feedback for non-existent room. Password is erased from memory upon entry.

## Type of Change

Please check the options that apply

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has the Changes Been Tested?

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] Changes included in this pull request covers minimal topic
- [ ] I have performed a self-review of my code
- [ ] I have commented my code properly, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by BrianYanGitHub** [comment]
Can we make it so that if room doesn't exist, the password checking doesn't happen? Right now it shows both errors. 

Confirm that the room check is working as intended. Good job

**Comment by BrianYanGitHub** [comment]
The errors also persist unexpectedly .

For example: Joining a non existing room. Then putting the correct room. Entering the room. Leaving the room. Error still persists

**Review by Eliza612 — COMMENTED** [review]
Looks good! Thanks for the fixing!
Just one question, when a user joined a room, quit, rejoin, and quit, during this process it seems to be a default `submit` with notification `Room doesn't exist`. Probably add `preventdefault` somewhere. I can approve the PR or not sure you wanna fix it in this PPR
image

File: app/src/components/left/Sidebar.tsx [hunk 1]
```diff
@@ -23,17 +23,6 @@ const Sidebar: React.FC<SidebarProps> = ({
...
-  // We decided to freeze so we've commented this and line 41 out
-
-  
-  // const handleTabClick = (event: React.MouseEvent, oldValue: number) => {
-  //   if (activeTab === oldValue) {
-  //     setActiveTab(null);
-  //     // toggleVisibility(false);
-  //   }
-  // };
-
```
Comment (Eliza612) [inline app/src/components/left/Sidebar.tsx:36]: probably add back

File: app/src/components/top/NavBar.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import MoreVertIcon from '@mui/icons-material/MoreVert';
 import NavBarButtons from './NavBarButtons';
 import NewExportButton from './NewExportButton';
 import { RootState } from '../../redux/store';
-import logo from '../../public/icons/win/logo.png';
+import logo from '../../public/icons/win/blue-R-white-bg.png';
```
Comment (Eliza612) [inline app/src/components/top/NavBar.tsx:9]: Probably replace in the future

**Review by Eliza612 — APPROVED** [review]
Thanks for the great work!

**Pull request merged**